roles/confluent.variables/defaults/main.yml

```diff
@@ -573,6 +599,9 @@ control_center_listener_hostname: "0.0.0.0"
 ### Boolean to configure Control Center with TLS Encryption. Also manages Java Keystore creation
 control_center_ssl_enabled: "{{ssl_enabled}}"
 
+### Only use to customize directory Control Center writes log files to. Default location is /var/log/confluent/control-center. NOTE- control_center.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that Control Center writes log files to. Default location is /var/log/confluent/control-center. NOTE- control_center.appender_log_path is deprecated.